vsce/server/src/server.ts

```diff
@@ -369,27 +380,36 @@ CallStack (from HasCallStack):
 		connection.console.log(`Found pattern: ${m}`);
 
 		// ERROR MESSAGE m: error: <json>
-		const errorJson = JSON.parse(m[0].substring(7));
+		const errorJson: ReachCompilerErrorJSON = JSON.parse(m[0].substring(7));
 
 		//connection.console.log(`Tokens: ` + tokens);
 		const linePos = errorJson.ce_position[0] - 1;
 		const charPos = errorJson.ce_position[1] - 1;
 		const suggestions = errorJson.ce_suggestions;
 		const actualMessage = errorJson.ce_errorMessage;
 		const offendingToken = errorJson.ce_offendingToken;
+		const REACH_ERROR_CODE = errorJson.ce_errorCode;
```

Comment:
My personal taste is that all caps is reserved for global things

Reply:
I see. I personally prefer it for all constants, but I think it's important for a codebase to have consistent code style.

Right now, my constant declaration is the one that's breaking consistency, so I renamed `REACH_ERROR_CODE` out of `CONSTANT_CASE` with f9ef7dd096ee871dc5d647c88632e36743964397.